ReactCommon/cxxreact/ModuleRegistry.cpp

```diff
@@ -83,7 +83,7 @@ folly::Optional<ModuleConfig> ModuleRegistry::getConfig(const std::string& name)
 
   // Initialize modulesByName_
   if (modulesByName_.empty() && !modules_.empty()) {
-    moduleNames();
+    moduleNamesVec_ = moduleNames();
```

Comment:
That's truly terrible that app crashes on the start. :(

However I don't see how this change can make any difference.

Calling `moduleNames();` here in the first place is very strange thing; I think the original intention was to initialize/warm `modulesByName_` but that's very obscure behavior which needs to be refactored anyway.

@yuweipei  Why do you believe that retaining a vector of strings can fix anything?